src/Orchard.Web/Modules/Orchard.Taxonomies/Services/TaxonomyService.cs

```diff
@@ -185,6 +185,12 @@ public class TaxonomyService : ITaxonomyService {
             return TermPart.Sort(result);
         }
 
+        public int GetTermsCount(int taxonomyId) {

```

Comment:
I don't think we need that with the feature. The only location where it's used we already have the full list loaded.